package.json

```diff
@@ -81,7 +81,8 @@
     "cypress": "13.8.1",
     "ws@8": "^8.17.1",
     "cookie@<0.7.0": ">=0.7.0",
-    "esbuild": "0.25.0"
+    "esbuild": "0.25.0",
+    "estree-util-value-to-estree@<3.3.3": ">=3.3.3"
```

Comment:
These overrides are getting out of hand. We should try to get rid of remix.

See also: https://github.com/iTwin/iTwinUI/pull/2447#discussion_r1978106520